community-extra-vars.yml

```diff
@@ -1,28 +1,28 @@
 acs:
   artifact_name: alfresco-content-services-community-distribution
   repository: "{{ nexus_repository.releases }}"
-  version: 23.1.0
+  version: 23.2.1
   edition: Community
 api_explorer:
   artifact_name: api-explorer
   repository: "{{ nexus_repository.releases }}"
-  version: 23.1.0
+  version: 23.2.0
 transform:
   artifact_name: alfresco-transform-core-aio
   repository: "{{ nexus_repository.releases }}"
-  version: 5.0.1
+  version: 5.1.0
 acc:
   artifact_name: alfresco-control-center
   repository: "{{ nexus_repository.releases }}"
   version: 8.3.0
 search:
   artifact_name: alfresco-search-services
   repository: "{{ nexus_repository.releases }}"
-  version: 2.0.8.2
+  version: 2.0.9.1
 amps:
   aos_module:
     repository: "{{ nexus_repository.releases }}/aos-module/alfresco-aos-module"
-    version: 2.0.0
+    version: 3.0.0
   googledrive_repo:
     repository: "{{ nexus_repository.releases }}/integrations/alfresco-googledrive-repo-community"
     version: 4.0.0
```

Comment:
should be 4.1
Also looks like the share AMP seem to point to the repo one

Also bump in `group_vars/all.yml`

Reply:
>  Also looks like the share AMP seem to point to the repo one

aren't they following the same release cadence?

Reply:
They do yes but the artifacts name for share is also `alfresco-googledrive-repo-community`